File: MyLanguageUtils/src/main/java/ru/herooo/mylanguageutils/file/filesize/FileSizeUnits.java
package ru.herooo.mylanguageutils.file.filesize;

public enum FileSizeUnits {
    KB ("KB", 1024),
    MB ("MB", KB.NUMBER_OF_BYTES * 1024)
    ;


    final String TITLE;
    final long NUMBER_OF_BYTES;
    FileSizeUnits(String title, long numberOfBytes) {
        this.TITLE = title;
        this.NUMBER_OF_BYTES = numberOfBytes;
    }
}

File: MyLanguageWeb/src/main/resources/static/js/classes/html/input/file/mime_type_with_size/file_size/file_size_units.js
export class FileSizeUnits {
    KB = new FileSizeUnit("KB", 1024);
    MB = new FileSizeUnit("MB", this.KB.NUMBER_OF_BYTES * 1024);
}

class FileSizeUnit {
    TITLE;
    NUMBER_OF_BYTES;

    constructor(title, numberOfBytes) {
        this.TITLE = title;
        this.NUMBER_OF_BYTES = numberOfBytes;
    }
}